src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames';
+import { Link, useLocation, useResolvedPath } from 'react-router-dom';
+import { Person } from '../../types';
+
+interface Props {
+  persone: Person | null,
```

Comment:
Don't render this component if you don't have the person
```suggestion
  person: Person,
```